### PR TITLE
Bug fix/test shell deadlocks timecritical

### PR DESCRIPTION
**Pull request by ObiWahn**

### CODE REVIEW

File: tests/js/common/shell/shell-deadlocks-timecritical-noncluster-mmfiles-grey.js [hunk 1]
```diff
@@ -70,36 +75,41 @@ function DeadlockSuite () {
...
-            db.UnitTestsDeadlock2.any();
-            db.UnitTestsDeadlock1.insert({ done: true });
-          }
-        });
-      } catch (err) {
-        // nothing to do here
+
+      // wait until task is started
+      while(db.UnitTestsDeadlock3.count() === 0) {
+        internal.sleep(1);
```
Comment (dothebart) [inline tests/js/common/shell/shell-deadlocks-timecritical-noncluster-mmfiles-grey.js]: rather sleep for 0.1

**Comment by ObiWahn** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7446/

**Comment by ObiWahn** [comment]
https://github.com/arangodb/planning/issues/4682

**Comment by jsteemann** [comment]
Started another PR run:
https://172.16.10.101/view/PR/job/arangodb-matrix-pr/7508/

**Comment by ObiWahn** [comment]
https://172.16.10.101/view/PR/job/arangodb-matrix-pr/7512/

**Comment by ObiWahn** [comment]
https://172.16.10.101/job/arangodb-matrix-pr-linux/

**Comment by ObiWahn** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7639/

**Comment by jsteemann** [comment]
Obsolete now.

**Pull request closed without merging**